server/chat-commands/room-settings.ts

```diff
@@ -175,6 +175,10 @@ export const commands: ChatCommands = {
 			if (prefix) {
 				return this.errorReply(`This battle is required to be public due to a player having a name prefixed by '${prefix}'.`);
 			}
+			if (room.battle.inviteOnlySetter && !user.can('mute', null, room) && room.battle.inviteOnlySetter !== user.id) {
+				return this.errorReply(`Only the person who set this battle to be invite-only can turn it off.`);
+			}
+			room.battle.inviteOnlySetter = user.can('mute', null, room) ? 'staff' : user.id;
```

Comment:
```suggestion
			room.battle.inviteOnlySetter = user.id;
```

Is the `staff` key necessary here? `user.id` should be sufficient, right?

Reply:
I added it because battle timers use it, but I can't think of anything that removing it would break. Let's try without it?